.env

```
@@ -0,0 +1,7 @@
+# --- MySQL ---
+MYSQL_DATABASE=spring
+MYSQL_USER=user
+MYSQL_PASSWORD=secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MYSQL_PASSWORD key should go before the MYSQL_USER key

(UnorderedKey)